### ISSUE TITLE
RECORD IS VARYING IN SIZE FROM 1 TO 9992 CHARACTERS ...

### ISSUE THREAD

**Issue by Reinhard-Prehofer** [uwol/cobol85parser#26]
Referring to attached source.
The record is varying clause not yet fully implemented ...
[ISSUE27.CBL.txt](https://github.com/uwol/cobol85parser/files/1190290/ISSUE27.CBL.txt)

```
Preprocessing file ISSUE25.CBL.
Preprocessing file ISSUE25.CBL.
Parsing file ISSUE25.CBL.
Collecting units in file ISSUE25.CBL.
Analyzing program units of compilation unit ISSUE25.
call to unknown data element ein-STAT
call to unknown data element ein-STAT
call to unknown data element ein-STAT
call to unknown data element ein-STAT
call to unknown data element ein-STAT
call to unknown data element ein-STAT
call to unknown data element ein-STAT
call to unknown data element ein-STAT
Exception in thread "main" java.lang.NullPointerException
	at io.proleap.cobol.asg.metamodel.impl.ProgramUnitElementImpl.createIntegerLiteral(ProgramUnitElementImpl.java:816)
	at io.proleap.cobol.asg.metamodel.data.file.impl.FileDescriptionEntryImpl.addRecordContainsClause(FileDescriptionEntryImpl.java:347)
	at io.proleap.cobol.asg.metamodel.data.file.impl.FileSectionImpl.addFileDescriptionEntry(FileSectionImpl.java:82)
	at io.proleap.cobol.asg.metamodel.data.impl.DataDivisionImpl.addFileSection(DataDivisionImpl.java:144)
	at io.proleap.cobol.asg.metamodel.impl.ProgramUnitImpl.addDataDivision(ProgramUnitImpl.java:74)
	at io.proleap.cobol.asg.metamodel.impl.CompilationUnitImpl.addProgramUnit(CompilationUnitImpl.java:65)
	at io.proleap.cobol.asg.visitor.impl.CobolProgramUnitVisitorImpl.visitProgramUnit(CobolProgramUnitVisitorImpl.java:66)
	at io.proleap.cobol.asg.visitor.impl.CobolProgramUnitVisitorImpl.visitProgramUnit(CobolProgramUnitVisitorImpl.java:1)
	at io.proleap.cobol.Cobol85Parser$ProgramUnitContext.accept(Cobol85Parser.java:842)
	at org.antlr.v4.runtime.tree.AbstractParseTreeVisitor.visitChildren(AbstractParseTreeVisitor.java:46)
	at io.proleap.cobol.Cobol85BaseVisitor.visitCompilationUnit(Cobol85BaseVisitor.java:27)
	at io.proleap.cobol.Cobol85Parser$CompilationUnitContext.accept(Cobol85Parser.java:764)
	at org.antlr.v4.runtime.tree.AbstractParseTreeVisitor.visit(AbstractParseTreeVisitor.java:18)
	at io.proleap.cobol.asg.visitor.impl.AbstractCobolParserVisitorImpl.visit(AbstractCobolParserVisitorImpl.java:1)
	at io.proleap.cobol.asg.runner.impl.CobolParserRunnerImpl.analyzeProgramUnits(CobolParserRunnerImpl.java:151)
	at io.proleap.cobol.asg.runner.impl.CobolParserRunnerImpl.analyze(CobolParserRunnerImpl.java:50)
	at io.proleap.cobol.asg.runner.impl.CobolParserRunnerImpl.analyzeFile(CobolParserRunnerImpl.java:92)
	at com.csi.parser.CobolParseDVZ.parseFile(CobolParseDVZ.java:60)
	at com.csi.parser.CobolParseDVZ.main(CobolParseDVZ.java:95)


```

**Comment by uwol** [comment]
Fixed on 16 Sep in a6c4b1b9f34fa1bd0e52b3b0f46ecde24282bf86.

**Comment by uwol** [comment]
Tested with ISSUE27.CBL.txt, which passes now.
